controller/handler/dashboard.js

```diff
@@ -15,41 +15,83 @@ const database = require('../../model');
  * @returns {View} Rendered page
  */
 function dashboardView (request, reply) {
-    const trial = database.sequelize.model('trial');
+    const todayDate = moment();
 
-    trial
-    .findAll()
-    .then((trials) => {
-        // Process data into format expected in view
-        const trialData = trials.map(processTrial);
+    return Promise
+        .all([
+            database.sequelize.query(
```

Comment:
@ChristianMurphy I wrote it like this because in future we might need to support multiple queries on dashboard view (for example to fetch data for radar chart). I can change it if you want.